stripe/src/main/java/com/stripe/android/view/CardFormView.kt

```diff
@@ -0,0 +1,419 @@
+            )
+            editText.setTextColor(
+                ContextCompat.getColor(
+                    context,
+                    R.color.stripe_card_form_view_textinput_color
+                )
+            )
+
+            editText.setBackgroundResource(android.R.color.transparent)
+
+            editText.setErrorColor(
+                ContextCompat.getColor(context, R.color.stripe_card_form_view_form_error)
+            )
+        }
+
+        cardMultilineWidget.expiryDateEditText.includeSeparatorGaps = true
+        cardMultilineWidget.expirationDatePlaceholderRes = null
+        cardMultilineWidget.expiryTextInputLayout.hint =
+            context.getString(R.string.stripe_paymentsheet_expiration_date_hint)
+        cardMultilineWidget.cardNumberTextInputLayout.placeholderText = null
+        cardMultilineWidget.setCvcPlaceholderText("")
+
+        cardMultilineWidget.cvcEditText.imeOptions = EditorInfo.IME_ACTION_NEXT
+        cardMultilineWidget.setBackgroundResource(R.drawable.stripe_card_form_view_text_input_layout_background)
+
+        val layoutMarginHorizontal =
+            resources.getDimensionPixelSize(R.dimen.stripe_card_form_view_text_margin_horizontal)
+        val layoutMarginVertical =
+            resources.getDimensionPixelSize(R.dimen.stripe_card_form_view_text_margin_vertical)
+        setOf(
+            cardMultilineWidget.cardNumberTextInputLayout,
+            cardMultilineWidget.expiryTextInputLayout,
+            cardMultilineWidget.cvcInputLayout
+        ).forEach { layout ->
+            layout.updateLayoutParams<LinearLayout.LayoutParams> {
+                marginStart = layoutMarginHorizontal
+                marginEnd = layoutMarginHorizontal
+                topMargin = layoutMarginVertical
+                bottomMargin = layoutMarginVertical
+            }
+            layout.isErrorEnabled = false
+            layout.error = null
+        }
+
+        cardMultilineWidget.setCvcIcon(R.drawable.stripe_ic_paymentsheet_cvc)
+
+        cardMultilineWidget.cardBrandIconSupplier =
+            CardMultilineWidget.CardBrandIconSupplier { cardBrand ->
+                CardMultilineWidget.CardBrandIcon(
+                    when (cardBrand) {
+                        CardBrand.Visa -> R.drawable.stripe_ic_paymentsheet_card_visa
+                        CardBrand.AmericanExpress -> R.drawable.stripe_ic_paymentsheet_card_amex
+                        CardBrand.Discover -> R.drawable.stripe_ic_paymentsheet_card_discover
+                        CardBrand.JCB -> R.drawable.stripe_ic_paymentsheet_card_jcb
+                        CardBrand.DinersClub -> R.drawable.stripe_ic_paymentsheet_card_dinersclub
+                        CardBrand.MasterCard -> R.drawable.stripe_ic_paymentsheet_card_mastercard
+                        CardBrand.UnionPay -> R.drawable.stripe_ic_paymentsheet_card_unionpay
+                        CardBrand.Unknown -> R.drawable.stripe_ic_paymentsheet_card_unknown
+                    }
```

Comment:
I believe we have this logic in a few places. Can we move it out to a separate class?

Reply:
Sure, will do it in a separate ticket as this PR is getting pretty big